specification.markdown

```diff
@@ -869,18 +869,18 @@ An EBML Reader may discard some or all data if the following errors are found in
 
 - Invalid Element Data Size values (e.g. extending the length of the EBML Element beyond the scope of the Parent Element; possibly triggering access-out-of-bounds issues).
 - Very high lengths in order to force out-of-memory situations resulting in a denial of service, access-out-of-bounds issues etc.
-- Missing EBML Elements that are mandatory and have no declared default value.
+- Missing EBML Elements that are mandatory in a Master Element and have no declared default value, making the semantic invalid at that Master Element level.
 - Usage of invalid UTF-8 encoding in EBML Elements of UTF-8 type (e.g. in order to trigger access-out-of-bounds or buffer overflow issues).
-- Usage of invalid data in EBML Elements with a date type.
+- Usage of invalid data in EBML Elements with a date type, trigerring bogus date accesses.
 
 Side channel attacks could exploit:
 
-- The semantic equivalence of the same string stored in a String Element or UTF-8 Element with and without zero-bit padding.
-- The semantic equivalence of VINT_DATA within Element Data Size with two different lengths due to left-padding zero bits.
-- Data contained within a Master Element which is not itself part of an EBML Element.
-- Extraneous copies of Identically Recurring Element.
-- Copies of Identically Recurring Element within a Parent Element that contain invalid CRC-32 Elements.
-- Use of Void Elements.
+- The semantic equivalence of the same string stored in a String Element or UTF-8 Element with and without zero-bit padding, making comparison at the semantic level invalid.
+- The semantic equivalence of VINT_DATA within Element Data Size with two different lengths due to left-padding zero bits, making comparison at the semantic level invalid.
+- Data contained within a Master Element which is not itself part of an EBML Element can trigger incorrect parsing behavior in EBML Readers.
```

Comment:
All data within a Master Element is part of an EBML Element (the same referenced Master Element). I suggest `which is not itself part of a Child Element`.

Reply:
OK